prow/config.yaml

```diff
@@ -18,36 +18,36 @@ presubmits:
   - name: pull-cadvisor-e2e
     always_run: true
     context: pull-cadvisor-e2e
-    rerun_command: "@k8s-bot pull-cadvisor-e2e test this"
-    trigger: "@k8s-bot (pull-cadvisor-e2e )?test this"
+    rerun_command: "/test pull-cadvisor-e2e"
+    trigger: "(^@k8s-bot (pull-cadvisor-e2e )?test this\r?$|^/test( pull-cadvisor-e2e)?\r?$)"
```

Comment:
The new regex is: 
`((?m)^@k8s-bot (pull-cadvisor-e2e )?test this,?(\\s+|$)|(?m)^/test( all| pull-cadvisor-e2e),?(\\s+|$))`

It is long, but IMO once we retired the old commands, it is readable.